package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "passcryptum",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "run-p type-check build-only",
+    "preview": "vite preview",
+    "test:unit": "vitest --environment jsdom --root src/",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore"
```

Много лишних расширений файлов для команды `lint`. Предлагаю оставить так.

```suggestion
    "lint": "eslint . --ext .vue,.js,.cjs,.ts --fix --ignore-path .gitignore"
```